src/Data/YAML.hs

```diff
@@ -54,8 +54,14 @@
 --
 module Data.YAML
     (
+      -- * Typeclass-based dumping
+      ToYAML(..)
+    , dumpYAML
+    , dumpYAML'
+    , dumpEvents
```

Comment:
Why is `dumpEvents` exported from this module? The `Data.YAML` module currently does not expose any operation which refers to `Events`

Reply:
ok, will remove that after the debugging process is over